orangecanvas/scheme/signalmanager.py

```diff
@@ -338,7 +338,7 @@ def _set_runtime_state(self, state):
         """
         state = SignalManager.RuntimeState(state)
         if self.__runtime_state != state:
-            self.__runtime_state = state
+            self.__runtime_state = SignalManager.RuntimeState(state)
```

Comment:
I'm curious: why another copy?

Reply:
Part of this was cherry picked from an older branch of mine. But since then this was was already fixed in https://github.com/biolab/orange-canvas-core/commit/c33d6195b45e735f0c94ad8a0bb6e67ad1557155. I removed the double 'fix'.